db/seeds.rb

```diff
@@ -132,3 +141,44 @@
 
   user.organisations << organisation
 end
+
+access_requester_user = User.create!(
+  first_name: Faker::Name.first_name,
+  last_name: Faker::Name.last_name,
+  email: Faker::Internet.email,
+  welcome_email_date_utc: 7.days.ago,
+  sign_in_user_id: SecureRandom.uuid,
+)
+
+Organisation.create!(
+  name: 'Acme',
+  org_id: '12345',
+  providers: [
+    accrediting_provider,
+    provider2,
+  ],
+  users: [
+    access_requester_user,
+    admin_user,
+  ],
+)
```

Comment:
Can we simplify this a little? I think that makes the seeds.rb file harder to read and reason about, it would be nice to just use the already-created users and providers.